src/SAWScript/X86.hs

```diff
@@ -657,17 +659,23 @@ x86 = x86_64MacawSymbolicFns
 --------------------------------------------------------------------------------
 -- Errors
 
-data X86Unsupported = X86Unsupported String deriving Show
-data X86Error       = X86Error String deriving Show
+-- | Exception for hitting an unsupported object or feature.
+data X86Unsupported = X86Unsupported FilePath String deriving Show
+
+-- | Exception for miscellaneous errors during verification.
+data X86Error       = X86Error FilePath (Maybe String) String deriving Show
```

Comment:
Now that these exception types each have multiple fields, it would be helpful to leave some Haddocks describing what each field represents. In particular, it's not obvious to me what the difference between the `Maybe String` and `String` fields of `X86Error` represent without looking at how `X86Error` is used elsewhere in the code.

Reply:
Added in ac7ecb3943784f624c619d78d09dc6a0a7ab0474.